src/components/Settlements/SettlementsHistory.vue

```diff
@@ -7,20 +7,20 @@
     <el-table-column type="expand">
       <template slot-scope="scope">
         <p>
-          {{ scope.row.from }} wanted to exchange {{ scope.row.offer_amount + ' ' + scope.row.offer_asset}} for {{ scope.row.request_amount + ' ' + scope.row.request_asset}} with {{ scope.row.to }}
+          {{ `${scope.row.from} wanted to exchange ${scope.row.offer_amount} ${scope.row.offer_asset} for ${scope.row.request_amount} ${scope.row.request_asset} with ${scope.row.to}` }}
```

Comment:
I prefer:
```
{{ var1 }} blah blah {{ var2 }}
```
than:
```
{{ `${var1} blah blah ${var2}` }}
```
However, since it is clear in either style that which string is a variable and which string is a constant, so I think either is fine unless the consistency over the whole repository is broken.

Since there are two styles mixed, I want you to fix it to either style if there is no special reason :)

Reply:
Ok!